adafruit_rfm69.py

```diff
@@ -597,7 +609,7 @@ def frequency_mhz(self, val):
         self._write_u8(_REG_FRF_LSB, lsb)
 
     @property
-    def encryption_key(self):
+    def encryption_key(self) -> bytearray:
```

Comment:
I think this may be `bytes` instead of `bytearray`. It's typed that way in the init function. 

There is one more that can be changed in the setter below here as well.

Reply:
Ah, nevermind actually. On second look I see it's making the bytearray that is returned.